### ISSUE TITLE
German Translation

### ISSUE THREAD

**Issue by avecsr** [IrcDirk/Carbonite-Classic#68]
Hey IrcDirk,

i found some phrases that were not that correct german. Can i help you translate some / all phrases?

**Comment by IrcDirk** [comment]
Of course, every one can help :) Just make fork and then create pull request.

**Comment by IrcDirk** [comment]
So edit files and send them to my email.
